src/tqec/compile/graph.py

```diff
@@ -464,14 +469,30 @@ def generate_stim_circuit(
                 Detectors are not computed and added to the circuit if this
                 argument is negative.
             detector_database: an instance to retrieve from / store in detectors
-                that are computed as part of the circuit generation.
+                that are computed as part of the circuit generation. If not given,
+                the detectors are retrieved from/stored in the provided
+                ``database_path``.
+            database_path: specify where to save to after the calculation, when augmented.
+                This defaults to tqec.utils.paths.DEFAULT_DETECTOR_DATABASE_PATH if
```

Comment:
I think that `:data` would be better:

```suggestion
                This defaults to :data:`.DEFAULT_DETECTOR_DATABASE_PATH` if
```
I'll commit that change.